src/main/java/teammates/ui/template/CourseEditInstructorPanel.java

```diff
@@ -17,6 +17,7 @@
     private ElementTag resendInviteButton;
     private ElementTag editButton;
     private ElementTag cancelButton;
+    private ElementTag newInstructorCancelButton;
```

Comment:
I think you should keep the names of the button the same and in sync with the action it dictates in diff files... previously you used `cancelAddInstructorButton`, and in ur test u used the name `cancelAddNewInstructor` for the action...so I suggest u choose one name and stick to it. Same for all the relevant places. Pls reexamine ur code and change the names.